src/liballoc/tests/btree/map.rs

```diff
@@ -519,7 +528,7 @@ fn test_range_1000() {
     #[cfg(not(miri))] // Miri is too slow
     let size = 1000;
     #[cfg(miri)]
-    let size = 144; // to obtain height 3 tree (having edges to both kinds of nodes)
+    let size = MIN_INSERTS_HEIGHT_2;
```

Comment:
Ah, this unfortunately broke the build in Miri as the type is now `usize` instead of `u32`... see https://github.com/rust-lang/rust/pull/70559

Reply:
I also forgot to check Linux builds and debug assertions lately. So I'll just push here, I guess.

Reply:
WDYM "push here"? This PR landed already...

Reply:
miri-test-libstd also enables debug assertions, so that part is covered (also I believe some CI runners have debug assertions enabled).

Reply:
Oops, I misinterpreted. Thanks for fixing.

I'm pretty sure the ordinary PR checks a few months ago, when there were 4 of them, did not include debug assertions, so I did those myself. And --no-doc to test benchmarks.

Reply:
Yeah the PR runner probably has debug assertions disabled, but some others at least used to have them enabled.